cypress/e2e/crypto/utils.ts

```diff
@@ -95,6 +95,10 @@ export function checkDeviceIsCrossSigned(): void {
             const myDeviceSignatures = res.body.device_keys[userId][myDeviceId].signatures[userId];
             expect(myDeviceSignatures[selfSigningKeyId]).to.exist;
         });
+
+    cy.findByRole("button", { name: "User menu" }).click();
+    cy.findByRole("menuitem", { name: "Security & Privacy" }).click();
+    cy.findByRole("button", { name: "Restore from Backup" }).should("exist");
```

Comment:
this seems like it doesn't really fit with the name or the documentation of the function. It's very unclear to someone looking at any of the places where `checkDeviceIsCrossSigned` is called that this will be done.

I'd suggest sticking it in its own function and adding calls in the tests.